packages/flutter/lib/src/widgets/banner.dart

```diff
@@ -236,7 +264,9 @@ class CheckedModeBanner extends StatelessWidget {
       result = new Banner(
         child: result,
         message: 'SLOW MODE',
-        location: BannerLocation.topRight);
+        textDirection: TextDirection.ltr,
```

Comment:
Is this correct?  I guess you're saying we'll localize the direction and the message as one unit.  I would have assumed if the user is generally using RTL, that we'd just inherit that here.  I think the net result will be the same in this case.

Reply:
We discussed this and will handle Banner in a separate patch.